x-pack/legacy/plugins/lens/public/indexpattern_plugin/operations/definitions/date_histogram.tsx

```diff
@@ -208,7 +208,11 @@ export const dateHistogramOperation: OperationDefinition<DateHistogramIndexPatte
                   <EuiFlexItem>
                     <EuiFieldNumber
                       data-test-subj="lensDateHistogramValue"
-                      value={interval.value}
+                      value={
+                        typeof interval.value === 'number' || interval.value === ''
+                          ? interval.value
+                          : parseInt(interval.value, 10)
```

Comment:
Maybe not for an EUI update PR but is this really a good spot to do this?

If a non-number has gotten to this point, would it be better to throw an error rather the furthering the rabbit hole of chasing down bad data? Basically, this could lead to subtle bugs that cause people to make mistakes assuming the data is correct.

Reply:
So the options here are to cast `value` as `EuiFieldNumberProps['value']` or attempt to resolve any errors (like so).

Earlier in the script, `parseInterval` has already transformed `interval.value` into `number | ''` like we want. Elsewhere in this file, though, `interval.value` is expected to be a `string`.